src/i18n/eng.json

```
@@ -46,8 +46,8 @@
   "opening_soon": "Opening soon!",
   "for_distribution": "for distributing",
   "for_receiving": "for receiving",
-  "seeking_volunteers": "Seeking Volunteers",
-  "seeking_money": "Seeking Money",
+  "seeking_volunteers": "Volunteer here",
```

```suggestion
  "seeking_volunteers": "Volunteer Here",
```
probably want both `here`s in the same case

---

haha whoops, good catch.